adapters/repos/db/lsmkv/memtable.go

```diff
@@ -52,7 +53,7 @@ func newMemtable(path string, strategy string,
 		path:             path,
 		strategy:         strategy,
 		secondaryIndices: secondaryIndices,
-		lastWrite:        time.Now(),
```

Comment:
Shouldn't we keep the lastWrite anyway? Even if it is not used to flush, it could be used, at least for debugging purposes.

Reply:
Personally I'm not fan of dead code left just in case. We can always restore it if actually needed.